web/nginx/nginx.conf

```diff
@@ -0,0 +1,85 @@
+# https://nginxconfig.io
+# https://www.nginx.com/resources/wiki/start/topics/examples/full/
+
+user nginx;
+pid /run/nginx.pid;
+#worker_processes  1;
+worker_processes auto;
+worker_rlimit_nofile 1024;
+
+# Keeps the logs free of messages about not being able to bind().
+# #daemon     off;
+
+#include /etc/nginx/modules-enabled/*.conf;
+#load_module "modules/ngx_pagespeed.so";
+
+events {
+  multi_accept on;
+  #worker_connections 65535;
```

Comment:
are these comments or commented out code? Is it necessary?

Reply:
Codes.

They are not necessary.
I left them there for me to reference in the future.

Reply:
Ok then leave them in